model/maslin/models/maslin/m2x/ooapopulation/ooapopulation.xtuml

```diff
@@ -6461,12 +6461,17 @@ INSERT INTO O_TFR
 // not really a state but is "Ignore" or "Cannot_Happen"
 
 select any fromState related by sm_sm->SM_STATE[R501] where (selected.Name == param.startState);
+// Give preference to finding an inherited event (by overwriting a local find).
+// This allows a dubious possibility of having a local event with the
+// same name as an event in the supertype.
```

Comment:
Is there ever a case where param.object != "" and there is not a supertype poly? Would that be the case if the modeler fully qualified the event with its own containing domain name?